### PR TITLE
feat: Add rpms and debs support.

### PR DESCRIPTION
**Pull request by kuops**
### What this PR does / why we need it:

/kind feature

### Which issue(s) this PR fixes:

Fixes # [1486](https://github.com/kubesphere/kubekey/issues/1486)

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/bootstrap/os/tasks.go [hunk 1]
```diff
@@ -398,13 +398,28 @@ type InstallPackage struct {
...
+	os := release.(*osrelease.Data).ID
+	var pkg []string
+	switch strings.ToLower(os) {
+	case "ubuntu", "debian":
+		pkg = i.KubeConf.Cluster.System.Debs
+	case "centos", "rhel":
+		pkg = i.KubeConf.Cluster.System.Rpms
+	default:
+		return fmt.Errorf("unsupported operation system %s", os)
+	}
```
Comment (24sama) [inline pkg/bootstrap/os/tasks.go:414]: I guess that would not work when the os distribution is Kylin, fedora, or others. So I suggest using assert to check the repo type, and we can use this as follow:
```go
repo, ok := host.GetCache().Get("repo")
if !ok {
	return errors.New("get repo failed by host cache")
}
r := repo.(repository.Interface)

var pkg []string
if _, ok := r.(*repository.Debian); ok {
  pkg = i.KubeConf.Cluster.System.Debs
} else if _, ok := r.(*repository.RedhatPackageManager); ok {
  pkg = i.KubeConf.Cluster.System.Rpms
}
```

**Review by 24sama — COMMENTED** [review]
Thanks for your contributions! Here are some comments, PTAL!

File: pkg/bootstrap/os/tasks.go (hunk 1 above)
Comment (kuops) [reply pkg/bootstrap/os/tasks.go]: thanks for your prompt reply.

**Comment by 24sama** [comment]
Thanks again! And, it would be better if you can create a new PR to improve the [file](https://github.com/kubesphere/kubekey/blob/master/docs/config-example.md).

**Comment by harris1024** [comment]
OK, great. Thank you very much for your contribution. Our business needs to use this function

**Pull request merged**